src/main/java/com/onevizion/scmdb/DdlGenerator.java

```diff
@@ -383,4 +386,21 @@ private void deleteFilteredFiles(File fileDir, String fileName) {
     public void generateDllsForAllDbObjects() {
         generateDdls(ddlDao.extractAllDbObjectsWithoutDdl(), true);
     }
+
+    private String sortConstraintInScript(String ddlScript) {
+        Pattern pattern = Pattern.compile(CONSTRAINT_PATTERN, Pattern.MULTILINE);
```

Comment:
save `pattern` as constant, compiling is expensive operation

Reply:
Fixed